lib/susedistribution.pm

```diff
@@ -571,10 +571,15 @@ sub init_consoles {
             my $packed_ip = gethostbyname($s390_guest_fqdn);
             die "Failed to get host by name for '$s390_guest_fqdn' (on " . hostname . ")" unless $packed_ip;
             my $s390_guest_ip = inet_ntoa($packed_ip);
-            my $host_ip = get_var("AGAMA") ? 'hostip' : 'HostIP';
-            my $host_name = get_var("AGAMA") ? 'hostname' : 'Hostname';
-            $s390_params .= " $host_ip" . "=${s390_guest_ip}/${s390_guest_subnetmask}";
-            $s390_params .= " $host_name" . "=${s390_guest_hostname}";
+            unless (get_var("AGAMA")){
+                $s390_params .= " HostIP=${s390_guest_ip}/${s390_guest_subnetmask}";
+                $s390_params .= " Hostname" . "=${s390_guest_hostname}";
+            }else{
+                my $s390_gateway_ip = "10.145.10.254";
```

Comment:
could this be REPO_HOSTO but replacing the last part with 254 ?

Reply:
21160037a